packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TWalletInfo } from 'Types';
+import { Text, WalletCard } from '@deriv/components';
+import { useCurrencyConfig } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { getWalletCurrencyIcon } from '@deriv/utils';
+import wallet_description_mapper from 'Constants/wallet_description_mapper';
+
+type TAddWalletCard = {
+    wallet_info: React.PropsWithChildren<TWalletInfo>;
+};
+
+const AddWalletCard = observer(({ wallet_info }: TAddWalletCard) => {
```

Comment:
```suggestion
const AddWalletCard = observer(({ wallet_info }: React.PropsWithChildren<TWalletInfo>) => {
```
A lot easier to read this way